test/suite/test_rollback_to_stable10.py

```diff
@@ -271,12 +276,17 @@ def test_rollback_to_stable_prepare(self):
         else:
             self.conn.set_timestamp('stable_timestamp=' + timestamp_str(50))
 
-        # Here's the update operation we'll perform, encapsulated so we can easily retry
+        # Here's the update operations we'll perform, encapsulated so we can easily retry
         # it if we get a rollback. Rollbacks may occur when checkpoint is running.
-        def simple_update(cursor, key, value):
-            cursor.set_key(key)
-            cursor.set_value(value)
-            self.assertEquals(cursor.update(), 0)
+        def range_updates(session, cursor, ds, value, nrows, prepare_config):
```

Comment:
can we name it as `prepare_range_updates`? as this function prepares the transaction.

Reply:
Done.